spec.html

```diff
@@ -39463,6 +39220,30 @@ <h1>GeneratorYield ( _iterNextObj_ )</h1>
           1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _genContext_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-yield" aoid="Yield">
+        <h1>Yield (_iterNextObj_)</h1>
+        <p>The abstract operation Yield takes argument _iterNextObj_. It performs the following steps when called:</p>
+        <emu-alg>
+          1. Return ? GeneratorYield(CreateIterResultObject(_value, *false*)).
+        </emu-alg>
```

Comment:
```suggestion
        <h1>Yield ( _value_ )</h1>
        <p>The abstract operation Yield takes argument _value_. It performs the following steps when called:</p>
        <emu-alg>
          1. Let _generatorKind_ be ! GetGeneratorKind().
          1. If _generatorKind_ is ~async~, then return ? AsyncGeneratorYield(_value).
          1. Otherwise, return ? GeneratorYield(CreateIterResultObject(_value_, *false*)).
        </emu-alg>
```

Reply:
Looks like there's still a typo on step 2 here, missing `_` in `? AsyncGeneratorYield(_value).`